client/payment-details/dispute-details/index.tsx

```diff
@@ -4,22 +4,35 @@
  * External dependencies
  */
 import React from 'react';
+import moment from 'moment/moment';
```

Comment:
This may be causing the inflation of the `dist/index.js` filesize. 
```suggestion
import moment from 'moment';
```